monkey/infection_monkey/Pipfile

```diff
@@ -26,7 +26,7 @@ email-validator = "*"
 monkey-types = "*"
 monkeyevents = "*"
 monkeytoolbox = "*"
-monkey-agentpluginapi = "==0.4.0"
+monkey-agentpluginapi = "*"
```

Comment:
The agent reqiures at least v0.4.0 of monkey-agentpluginapi in order to function properly. Using "*" could allow lower versions to be installed, resulting in malfunctions. Instead, I think we should set this to ">=0.5.0"

Reply:
I'll drop this line as it wasn't intended to be part of the commit